include/ifc/abstract-sgraph.hxx

```diff
@@ -2891,20 +2901,17 @@ namespace ifc {
             using ValueType = T;
         };
 
-// Note: VC and the build settings do not want to see idiomatic C++ by default.
-#pragma warning(push)
-#pragma warning(disable : 4624)
+
```

Comment:
Hmm, don't remove the suppression for this warning for MSVC.

Reply:
[C4624](https://learn.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-1-c4624?view=msvc-170):

> 'derived class' : destructor was implicitly defined as deleted because a base class destructor is inaccessible or deleted

How is the warning relevant here? All the types and their bases here are structs, so the dtor is accessible everywhere.

Reply:
Hmm, that warning used to say something else, along the line that a base class didn't have a virtual destructor.   We need to double check that before we remove it, or it might cause build break on some other projects that depend on this header.

Reply:
I replaced the comment above with a FIXME instead.